docs/environments/atari/alien.md

```diff
@@ -55,6 +55,7 @@ You score points by destroying eggs, killing aliens, using pulsars, and collecti
 by an alien, you will lose one of your lives. The number of lives you have depends on the game flavor. For a
 table of scores corresponding to the different achievements, consult [the AtariAge page](https://atariage.com/manual_html_page.php?SoftwareID=815).
 
+
```

Comment:
Do you know why this new line is created?

Reply:
must have been an accident while rebuilding the dictionary and converting back to json. fixed!